src/services/jobs.js

```diff
@@ -0,0 +1,81 @@
+/**
+ *
+ * @module jobs
+ */
+
+import http from './http';
+
+/**
+ * Get active jobs for current user
+ * @param {String} details The user's details
+ * @return {Promise.<String>} User's active jobs
+ */
+const getActiveJobsForUser = details => {
+  return http.post(`jobs/getJobs`, details);
```

Comment:
How does http.post() work for retrieving data? I would expect a http.get().

Reply:
With `http.get()`, any data that we send would need to be in a URL parameter. We needed to send a DateTime object, and there wasn't a good way to make that into a URL parameter because it would contain a lot of characters that need to be sanitized if they're part of a url. `http.post()` allows you to send data as an object without sanitization, which is more ideal for this scenario.

Reply:
okay, and you get the active jobs back in response?